busted/runner.lua

```diff
@@ -52,7 +52,7 @@ return function(options)
 
   local luacov = require 'busted.modules.luacov'()
 
-  require 'busted.init'(busted)
+  package.loaded['busted'] = require 'busted.init'(busted)
```

Comment:
I don't think I agree with this solution. Why would you do it in `runner.lua`? Doesn't it make sense to do it in the module that it affects, `init.lua`. Personally I think what I suggested in the issue is a better approach. It should not be possible to load a copy of the same module.